### PR TITLE
CIP 4 and 5, Module Readiness Checklist and Committee

### LINKED ISSUE #4: CIP-4: Cosmos Hub Module Readiness Checklist
Trying the approach of creating an issue with a PR as it seems to me that it'll be easier to track CIP identifiers rather than searching through PRs and their commits.

### LINKED ISSUE #5: CIP-5: Module Readiness Committee
Depends on #4

### PR DESCRIPTION
**Pull request by shahankhatch**
Closes #4 
Closes #5

### CODE REVIEW

File: CIPS/cip-4.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Governance
+
+- [ ] Hub governance proposal
+- [ ] Core community governance
+- [ ] Steering community
+- [ ] Timelines and Roadmap
+
+Research
+
+- [ ] Working Group/Interest Group and Charter
```
Comment (hxrts) [inline CIPS/cip-4.md]:
```suggestion
- [ ] Working group/interest group and charter
```

File: CIPS/cip-4.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+# Module Readiness Checklist: `Module Name`
+
+`Summary`
+
+Overview
+
+- [ ] ADR
+- [ ] Specification
+- [ ] Documentation
+- [ ] Light Paper
```
Comment (hxrts) [inline CIPS/cip-4.md]:
```suggestion
- [ ] Light paper
```

File: CIPS/cip-4.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- [ ] Developer:
+- [ ] Maintainer:
+- [ ] Handoff requirements:
+- [ ] Support and training requirements:
+
+Integrations
+
+- [ ] Gaia integration
+- [ ] Integration partners:
+- [ ] IBC Readiness
```
Comment (hxrts) [inline CIPS/cip-4.md]:
```suggestion
- [ ] IBC readiness
```

File: CIPS/cip-4.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- [ ] Downstream user impact report
+- [ ] Upstream partner impact report
+- [ ] Inter-module dependence, Ethereum Full node, any potential NFT module on the Hub
```
Comment (hxrts) [inline CIPS/cip-4.md]:
```suggestion
- [ ] Inter-module dependence, Ethereum full node, any potential NFT module on the Hub
```

File: CIPS/cip-4.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+Governance
+
+- [ ] Hub governance proposal
+- [ ] Core community governance
+- [ ] Steering community
```
Comment (hxrts) [inline CIPS/cip-4.md]:
```suggestion
- [ ] Steering committee
```

File: CIPS/cip-4.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- [ ] Timelines and Roadmap
```
Comment (hxrts) [inline CIPS/cip-4.md]:
```suggestion
- [ ] Timeline and roadmap
```

**Comment by hxrts** [comment]
The research section is maybe a little vague, could be more actionable, but apart from that LGTM.

File: CIPS/cip-4.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+created: 2021-05-05
+---
+
+This document is used as a template for Module Readiness Checklists.
+
+It is intended to serve mainnet decision makers with relevant information for deciding when a module is ready for deployment onto a mainnet, e.g., Cosmos Hub.
+
+The checklist is intended to be easy to complete by the Module Readiness Committee.
+
+Not all items on the checklist need to have items.
```
Comment (ValarDragon) [inline CIPS/cip-4.md]:
```suggestion
Not all items on the checklist apply to every module, they should be removed based on context.
```

Is this what is meant?

Comment (shahankhatch) [reply CIPS/cip-4.md]: Yes, thanks.

**Pull request merged**